xbmc/cores/VideoPlayer/Edl.cpp

```diff
@@ -179,9 +180,9 @@ bool CEdl::ReadEdl(const std::string& mediaFilePath, float fps)
           {
             fieldParts[1] = fieldParts[1].substr(0, 3);
           }
-          editStartEnd[i] =
-              static_cast<int64_t>(StringUtils::TimeStringToSeconds(fieldParts[0])) * 1000 +
-              std::atoi(fieldParts[1].c_str()); // seconds to ms
+          editStartEnd[i] = std::chrono::duration_cast<std::chrono::milliseconds>(
+              std::chrono::seconds(StringUtils::TimeStringToSeconds(fieldParts[0])));
+          editStartEnd[i] += std::chrono::milliseconds(std::atoi(fieldParts[1].c_str()));
```

Comment:
Could use `std::stoi` to avoid the `c_str` call but may have to handle the possible exception.

Reply:
ended up using `std::from_chars` since it doesn't throw and leave the resulting value unmodified in case of errors. Seem to fit the implementation here better as we're appending to the already existing value